web/styles/subscriptions.css

```diff
@@ -944,8 +944,9 @@ h4.user_group_setting_subsection_title {
             justify-content: space-between;
             position: absolute;
             bottom: 0;
-            width: calc(100% - 27px);
-            padding: 9px 15px 15px;
+            /* Subtract 15px padding on either side. */
+            width: calc(100% - 30px);
```

Comment:
Probably not worth it at this point, but these kinds of calculations would be nice to express with more variables.